src/services/popover-orientation/popover-orientation.service.ts

```diff
@@ -28,7 +28,7 @@ export class PopoverOrientationService {
 export class PopoverOrientationListener {
 
     /** Allow subscribing to state changes */
-    orientation$ = new Subject<PopoverOrientation>();
+    orientation$ = new BehaviorSubject<PopoverOrientation>(null);
```

Comment:
Default orientation is down, so that would make a good initial value here.

Reply:
updated to down as the initial value